cookbooks/aws-parallelcluster-slurm/files/default/config_slurm/scripts/health_check_manager.py

```diff
@@ -0,0 +1,375 @@
+                        # not yet implemented
+                        pass
+                    else:
+                        health_check = HealthCheckDefinition(
+                            name=health_check_key,
+                            is_managed=True,
+                            is_enabled=bool(
+                                health_check_value.get("Enabled")
+                                if health_check_value.get("Enabled") is not None
+                                else queue["HealthChecks"][health_check_key].get("Enabled", False)
+                            ),
+                            check_path=ManagedHealthCheckName[health_check_key.upper()].get_health_check_path(
+                                health_check_manager_config.managed_health_check_dir
+                            ),
+                        )
+                        health_checks.append(health_check)
+
+        except (KeyError, TypeError) as err:
+            if hasattr(err, "message"):
+                err = err.message
+            log.warning("Ignore missing %s in config", err)
+
+        health_check_conf = HealthCheckConfig(
+            region=args.region,
+            stack_name=args.stack_name,
+            node_type=args.node_type,
+            queue_name=args.queue_name,
+            compute_resource_name=args.compute_resource_name,
+            health_checks=health_checks,
+        )
+
+        log.debug("HealthCheck config %s: ", health_check_conf)
+        return health_check_conf
+
+
+def _execute_health_checks(health_check_manager_config: HealthCheckManagerConfig, args: List[str]) -> int:
+    """Execute all Health Check."""
+    health_check_conf = HealthCheckConfigLoader().load_configuration(health_check_manager_config, args)
+
+    exit_code_sum = 0
+
+    for health_check in health_check_conf.health_checks:
+        if health_check.is_enabled:
+            try:
+                log.info(
+                    "Executing Health Check '%s' for queue '%s' and compute resource '%s'",
+                    health_check.name,
+                    health_check_conf.queue_name,
+                    health_check_conf.compute_resource_name,
+                )
+
+                # The command in this subprocess call is built as literal
+                result = subprocess.run(
+                    health_check.check_path,
+                    timeout=health_check_manager_config.health_check_timeout,
+                    stdout=subprocess.PIPE,
+                    stderr=subprocess.STDOUT,
+                    encoding="utf-8",
+                    check=False,
+                    shell=False,  # nosec B603
+                )
+                exit_code_sum += result.returncode
+                if result.stdout:
+                    log.info("Output of Health Check '%s' execution is:\n%s", health_check.name, result.stdout)
+                else:
+                    log.info("Output of Health Check '%s' execution is empty", health_check.name)
```

Comment:
minor: to avoid code duplication you can do:
```python
                if result.stdout:
                    output = f":\nresult.stdout"
                else:
                    output = " empty"
                log.info("Output of Health Check '%s' execution is%s", health_check.name, output)
```